ocdaction/profiles/models.py

```diff
@@ -11,3 +11,10 @@ class OCDActionUser(AbstractEmailUser):
     date_birth = models.DateField('date of birth', null=True, blank=True)
     have_ocd = models.BooleanField(default=True, blank=True)
 
+class Task(models.Model):
+    taskname = models.CharField(max_length=100, blank=True)
```

Comment:
can we make the task name mandatory? Currently it will allow it to be save blank so we want to remove `blank=True` by default it is false